src/equations/linearized_euler_2d.jl

```diff
@@ -143,6 +143,44 @@ end
     end
 end
 
+# Calculate estimate for minimum and maximum wave speeds for HLL-type fluxes
+@inline function min_max_speed_naive(u_ll, u_rr, orientation::Integer,
+                                     equations::LinearizedEulerEquations2D)
+    min_max_speed_davis(u_ll, u_rr, orientation, equations)
+end
+
+@inline function min_max_speed_naive(u_ll, u_rr, normal_direction::AbstractVector,
+                                     equations::LinearizedEulerEquations2D)
+    min_max_speed_davis(u_ll, u_rr, normal_direction, equations)
+end
```

Comment:
That's what I would consider as default, too. Thus, I wonder whether we really need the current version of `min_max_speed_naive` or whether we should basically deprecate it in favor of `min_max_speed_davis`.